packages/scene-graph-mediator/runtime/src/importer/Importer.ts

```diff
@@ -211,4 +216,15 @@ export abstract class Importer {
 
     return objectMap;
   }
+
+  protected createRuntimeObjectForPlugins(node: Node, resources: any): any | null {
```

Comment:
```ts
protected createRuntimeObjectForPlugins<TR extends object, T1 extends object>(node: Node, resources: T1): TR | null {
```

If you don't wanna use 'any', you can avoid it by this code.

Reply:
It will be helpful.
However, I matched the caller and the definition, I think it is better not to change here.

Reply:
sure!